src/app/book/[id]/page.tsx

```diff
@@ -0,0 +1,120 @@
+import { Suspense } from "react";
+
+import { books } from "@/mock/books";
+import { notFound } from "next/navigation";
+import Image from "next/image";
+
+import styles from "./page.module.css";
+import MingcuteComments from "@/icons/MingcuteComments";
+import MingcuteUser from "@/icons/MingcuteUser";
+import MingcuteStarFill from "@/icons/MingcuteStarFill";
+import { BookModel } from "@/types/models/book.models";
+import BrifeComponent from "./brife.component";
+
+type Props = {
+  params: { id: string };
+};
+
+export default async function Page({ params }: Props) {
+  const book = await getBook(params?.id);
+
+  if (!book) {
+    return notFound();
+  }
+
+  const renderStars = (stars: number) => {
+    return Array.from({ length: stars }, (_, index) => (
+      <MingcuteStarFill key={index} />
+    ));
+  };
+
+  const FormattedDate = (date: string | Date | undefined): string => {
+    if (!date) return "";
+
+    const dateObj = typeof date === "string" ? new Date(date) : date;
+
+    return new Intl.DateTimeFormat("en-US", {
+      year: "numeric",
+      month: "2-digit",
+      day: "2-digit",
+    }).format(dateObj);
+  };
+
+  return (
+    <div className={styles["each-book-container"]}>
+      <div className={styles["top-section"]}>
+        <div className={styles["each-image"]}>
+          <Image src={book?.image} alt={book?.name} width={400} height={400} />
+        </div>
+
+        <div className={styles["each-description-container"]}>
+          <div className={styles["in-stock-label"]}>
+            {book?.inStock == true ? <span>موجود</span> : <span>ناموجود</span>}
+          </div>
+          <h1>{book?.name}</h1>
+          <div className={styles.author}>
+            <div>
+              <span className={styles.bold}>نویسنده: </span>
+              <span> {book?.author}</span>
+            </div>
+            <div>
+              <span className={styles.bold}>امتیاز: </span>
+              <span>{book?.rating}</span>
+            </div>
+          </div>
+          <div className={styles["each-price"]}>{book?.price} تومان</div>
```

Comment:
از toLocaleString استفاده کنید که سه رقم سه رقم جدا کنه که راحت‌تر بشه خوندنش

```suggestion
          <div className={styles["each-price"]}>{book?.price.toLocaleString()} تومان</div>
```